tests/unit/lib/SuiteCRM/Utility/OperatingSystemTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+
+class OperatingSystemTest extends \Codeception\Test\Unit
+{
+    use \SuiteCRM\StateCheckerTrait;
```

Comment:
can you just replace \Codeception\Test\Unit => SuiteCRM\StateCheckerUnitAbstract